client/src/components/Project/ChangeItems.js

```diff
@@ -3,30 +3,37 @@
  */
 
 import React, {Component} from 'react';
-import {Button, Panel, Table} from 'react-bootstrap';
+import {Button, Glyphicon, Panel, Table} from 'react-bootstrap';
+import {LinkContainer} from 'react-router-bootstrap';
 
 export default class ChangeItems extends Component {
+
     render() {
         return (
-            <Panel header={<h3 className="text-center">Attached Change Items</h3>}>
+            <Panel header="Attached Change Items">
                 <Table striped hover responsive>
                     <thead>
                     <tr>
-                        <th>View</th>
+                        <th/>
                         <th>Name</th>
                         <th>Status</th>
                         <th>LID</th>
                     </tr>
                     </thead>
                     <tbody>
-                    <tr>
-                        <td>
-                            <Button href="/project/:projectCode/:changeItem" bsStyle="success">View</Button>
-                        </td>
-                        <td>Placeholder</td>
-                        <td>Placeholder</td>
-                        <td>DD/MM/YYYY</td>
-                    </tr>
+                    {this.props.changeItems.map(changeItem => {
```

Comment:
This should be seperate out into it's own component. Should nest components in React, best to keep it all modular